classes/various/Matrix.sc

```diff
@@ -10,15 +10,24 @@ Matrix[slot] : Array {
 	*newClear { arg rows=1, cols=1; // return (rows x cols) - zero matrix
 		^super.fill(rows, { Array.newClear(cols).fill(0) });
 	}
-	*with { arg array; // return matrix from 2D array (array of rows)
-		var rows;
-		if((array.flat == array.flop.flop.flat)
-		.and(array.flatten.every({arg element; element.isNumber})) ,{
+	/* Michael McCrea, 2019 */
+	// refactor to avoid .flop.flop, which if called on Array, will expose
+	// it to a bug that could fail on large arrays.
+	// also, this is more efficient
```

Comment:
@joslloand I think this comment could be removed, and added to the PR description. For reference, the bug to which this refers [(#4192)](https://github.com/supercollider/supercollider/pull/4192), which will hopefully be finally merged into `3.11`.